dotcom-rendering/src/web/components/Nav/ExpandedMenu/VeggieBurgerMenu.tsx

```diff
@@ -77,6 +77,7 @@ const veggieBurgerStyles = (display: ArticleDisplay) => css`
 	position: absolute;
 	border: 0;
 	border-radius: 50%;
+	z-index: 1;
```

Comment:
This change is to make sure the bottom of the burger menu does not get hidden. It addresses the following edge case: 
a. The first container is a thrasher. 
b. The front path starts with `news/series` or any other combination that does not open any submenu, e.g https://www.theguardian.com/news/series/pandora-papers. (As opposed to this, a path that opens the News submenu would be `uk-news/series`.) 

| Before | After |
|--------|-------|
| ![image](https://github.com/guardian/dotcom-rendering/assets/19683595/40406678-1218-42ef-9dc3-8069b26231ab) | ![image](https://github.com/guardian/dotcom-rendering/assets/19683595/8bad87de-5eaa-42f0-9289-49b4afb97a99) |

This problem does not occur when only condition _a_ is true, e.g. in https://www.theguardian.com/uk-news/series/cost-of-the-crown

![image](https://github.com/guardian/dotcom-rendering/assets/19683595/c9bee70f-f29a-439b-8f0c-22e5d9bc9096)